templates/index.html

```diff
@@ -1,93 +1,106 @@
-		<ul id= "logo">
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/logo.png')}}" alt="the state of music logo" height="100"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/logo2.png')}}" alt="the state of music logo" height="100"></a>
-			</li>
-		</ul>
-
-		<!-- icons to allow toggle between genres - hrefs to be updated -->
-		<ul id = 'navbar'>
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/pop.png')}}" alt="pop music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/indie.png')}}" alt="indie music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/country.png')}}" alt="country music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/classic.png')}}" alt="classic music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/hiphop.png')}}" alt="hiphop music icon" height="105"></a>
-			</li>
-
-			<li>
-				<a href=#>
-					<img src="{{url_for('static',filename = 'images/metal.png')}}" alt="metal music icon" height="105"></a>
-			</li>
-		</ul>
-
-		 <!--about section - text to be updated-->
-		<div id="displayInfo">
-     	<div>
-				<h3>About</h3>
-				<small>
-     			This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-					This is some stuff about music in the US.
-     	</div>
-     </div>
-
-		 <div id="displayStats">
-				 <div>
-				 <h3>These states love music:</h3>
-				 <small>
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 This is some stuff about music in the US.
-				 </div>
-		 </div>
+<body>
+<div id='container'>
+
+	<!--<div id = "cityMap"></div> div to hold map at city level -->
+
+	<div class="row" id = "navbar">
+		<!-- loading logos here -->
+		<div class="col-md-3" id="logo">
+			<a href=#>
+				<img src="{{url_for('static',filename = 'images/logo.png')}}" alt="the state of music logo" height="100">
+				<img src="{{url_for('static',filename = 'images/logo2.png')}}" alt="the state of music logo" height="100">
+			</a>
+		</div>
+		<!-- loading icons here -->
+		<div class="col-md-9">
+			<div class="row">
+				<div class="col-sm-2 col-md-2">
+					<a id = 'icon' href=#>
+					<img src="{{url_for('static',filename = 'images/pop.png')}}" alt="pop music icon" height="100"> </a>
```

Comment:
Oh boo - This comment didn't make it in the review for some reason. Anyway. By setting the height to 100, you are overwriting bootstrap's width for the column and setting the width always to 100px (because they are square images). 
Instead, if you set the width of the image to 100%, it will adjust to the width of its parent div (the column).

Screenshots:
<img width="915" alt="Screen Shot 2019-05-01 at 12 42 13 AM" src="https://user-images.githubusercontent.com/929246/57000362-2a9ca780-6bab-11e9-92f6-51c84ac4414f.png">
<img width="1263" alt="Screen Shot 2019-05-01 at 12 30 56 AM" src="https://user-images.githubusercontent.com/929246/57000371-2ff9f200-6bab-11e9-8345-d117c9d862b3.png">